x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -117,6 +118,62 @@ setup:
 
   - match: { fields.sparse_field.semantic_text.searchable: true }
 
+---
+"Field caps set include_empty_field to true with sparse embedding":
+
+  - requires:
+      cluster_features: "gte_v8.16.0"
+      reason: field_caps support for semantic_text added in 8.16.0
+
+  - do:
+      field_caps:
+        include_empty_fields: true
+        index: test-index
+        fields: "*"
```

Comment:
No need to add a new YAML test for this case. You can check this in the previous test before indexing a document.